services/movie.py

```diff
@@ -0,0 +1,39 @@
+import init_django_orm  # noqa: F401
+from django.db.models.query import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
```

Comment:
Optional[list[int]]
`genres_ids = None` it is already mean that it must accept optional
The same in all other places

Reply:
> Optional[list[int]] `genres_ids = None` it is already mean that it must accept optional The same in all other places

Can you explain please, Do I have to replace genres_ids: list[int] = None with Optional[list[int]]?

Reply:
> > Optional[list[int]] `genres_ids = None` it is already mean that it must accept optional The same in all other places
> 
> Can you explain please, Do I have to replace genres_ids: list[int] = None with Optional[list[int]]?

Yes